python-threatexchange/threatexchange/storage/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
You'll need to move this to its own PR, then we'll need to bump the version and release a new bugfix version!